app/src/main/java/me/soundlocker/soundlocker/ApplicationsList.java

```diff
@@ -26,6 +32,13 @@ public void onCreate(Bundle bundle) {
         applicationsNameList = buildApplicationsNameList(applicationsList);
         adapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, applicationsNameList);
         setListAdapter(adapter);
+
+        firstBoot = storage.getFirstBoot(this.getApplicationContext());
+        if (firstBoot == false){
+            masterId = new BigInteger(256, random).toString(32);
```

Comment:
@patrickshao Shouldn't this be storage.getFirstBoot(this.getApplicationContext(), true); ? I'm a bit confused since it looks inverted to me.